From 017ae5929312685c4c044765445bfd4d2a4abb5c Mon Sep 17 00:00:00 2001
From: Bruce Ashfield <bruce.ashfield@windriver.com>
Date: Mon, 3 Oct 2016 12:27:57 -0400
Subject: [PATCH] if_tunnel: remove include of if/ip/in6.h

commit 1fe8e0f074c [include/uapi/linux/if_tunnel.h: include linux/if.h, linux/ip.h and linux/in6.h]
breaks the builds of net-tools.

We remove the new includes until such a time that userspace can adapt to the
new kernel headers.

Signed-off-by: Bruce Ashfield <bruce.ashfield@windriver.com>
---
 include/uapi/linux/if_tunnel.h | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/include/uapi/linux/if_tunnel.h b/include/uapi/linux/if_tunnel.h
index 777b6cd..1046f55 100644
--- a/include/uapi/linux/if_tunnel.h
+++ b/include/uapi/linux/if_tunnel.h
@@ -2,9 +2,6 @@
 #define _UAPI_IF_TUNNEL_H_
 
 #include <linux/types.h>
-#include <linux/if.h>
-#include <linux/ip.h>
-#include <linux/in6.h>
 #include <asm/byteorder.h>
 
 
-- 
2.7.4

